backend/app/Http/Controllers/StudentController.php

```diff
@@ -14,17 +14,17 @@ public function store(Request $request)
       'name' => 'required|string|max:255',
       'username' => 'required|string|max:255',
       'email' => 'required|string|unique:students,email|max:255',
-      'thumbnail' => 'required|image|max:255',
+      'thumbnail' => 'nullable|image|max:255',
       'password' => 'required|string|confirmed|max:255'
     ]);
 
-    $fields['thumbnail'] = request()->file('thumbnail')->store('thumbnails');
-
+    // $fields['thumbnail'] = request()->file('thumbnail')->store('thumbnails');
```

Comment:
put extra comments why you commented this out so that next developer can understand why and why it was commented out

Reply:
Got it Sir, currently working on it.